src/main/java/parkinglot/ParkingLot.java

```diff
@@ -0,0 +1,131 @@
+package parkinglot;
+
+public class ParkingLot {
+
+    private String name;
+    private String address;
+    private int capacity;
+
+    private int freeSpots = 0;
+
+    private ParkingSpot[] parkingSpots;
+    private ParkingTicket[] parkingTickets;
+    private Vehicle[] vehicles;
+
+
+    // initiate Parking Lot
+    ParkingLot(int capacity) {
+
+        //System.out.println("Initialization of new parking lot with capacity: " + capacity);
+        this.capacity = capacity;
+
+        parkingSpots = new ParkingSpot[capacity];
+        parkingTickets = new ParkingTicket[capacity];
+        vehicles = new Vehicle[capacity];
+
+        for (int i = 0; i < capacity; i++) {
+            parkingSpots[i] = new ParkingSpot(i + 1);
+            freeSpots++;
+        }
+    }
+
+    private boolean isFull () {
+        if (freeSpots == 0) {
+            return true;
+        }
+        return false;
```

Comment:
you may simply return freeSpots == 0;
result is the same but less code

Reply:
I agree. Will be corrected in next task

Reply:
Corrected. https://github.com/AqaCoreSchool/homeworks/pull/28/commits/57099748843c75dfa4f31fcca940318807f1aa55